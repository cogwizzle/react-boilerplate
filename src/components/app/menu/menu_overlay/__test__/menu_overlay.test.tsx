import * as React from 'react'
import MenuOverlay from '../menu_overlay'
import * as renderer from 'react-test-renderer'

describe('Render menu overlay', () => {
  it('isOpen', () => {
    const component = renderer.create(
      <MenuOverlay isOpen={true} />
    )
    const results = component.toJSON()
    expect(results).toMatchSnapshot()
  })

  it('Not isOpen', () => {
    const component = renderer.create(
      <MenuOverlay isOpen={false} />
    )
    const results = component.toJSON()
    expect(results).toMatchSnapshot()
  })
})

