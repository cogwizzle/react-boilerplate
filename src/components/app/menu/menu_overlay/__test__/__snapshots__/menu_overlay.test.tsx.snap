// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render menu overlay Not isOpen 1`] = `null`;

exports[`Render menu overlay isOpen 1`] = `
<div
  className="menu_overlay"
/>
`;
